ci-operator/config/operator-framework/community-operators/operator-framework-community-operators-master.yaml

```diff
@@ -0,0 +1,33 @@
+base_images:
+  cli:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
+build_root:
+  project_image:
+    dockerfile_path: scripts/ci/Dockerfile.ci-operator
+resources:
+  '*':
+    requests:
+      cpu: 1000m
+      memory: 4Gi
+tag_specification:
+  name: "4.6"
+  namespace: ocp
+tests:
+- as: deploy-operator-on-openshift
+  commands: |
+    echo "Need to clone default branch, cloning..."
+    mkdir -p /tmp/oper
+    cd /tmp/oper
+    git clone https://github.com/J0zi/community-operators.git
+    cd community-operators
+    git checkout RHO-716-release-on-openshift
```

Comment:
Why is this needed? This is an anti-pattern that we discourage. Why does this need to depend on out-of-tree fork?

Reply:
@petr-muller the test is in beta and we suppose to update it frequently until we get stable version. Then we definitely remove this part, merge master and use it. But until then, I will craft my branch until satisfied. Let me know if it is possible until next PR.

Reply:
It's possible, but like you spotted, you will likely encounter artifact reuse if the only thing that changed between individual test runs is the out-of-tree code.